cspuz_rs/src/serializer.rs

```diff
@@ -1385,8 +1385,20 @@ pub struct KudamonoURLInfo<'a> {
     pub content: &'a str,
 }
 
+pub fn get_kudamono_url_body(url: &str) -> Option<&str> {
+    if url.find("pedros.works").is_none() {
+        None
+    } else if url.find("?").is_none() {
+        None
+    } else {
+        let i = url.find("?")? + 1;
+        let url = url.get(i..)?;
+        Some(url)
```

Comment:
`url.get(i..)` returns `Option<&str>`, so we can just use this as the return value